module/programs/helper.js

```diff
@@ -630,19 +702,25 @@ module.exports = class ProgramsHelper {
 	 * remove entities in program scope.
 	 * @method
 	 * @name removeEntitiesInScope
-	 * @param {String} programId - Program Id.
-	 * @param {Array} entities - entities.
-	 * @returns {JSON} - Removed entities data.
+	 * @param {Object} bodyData - body data.
+	 * @param {Object} userDetails - User Details
+	 * @param {Boolean} organizations - If organizations is Present.
+	 * @returns {JSON} - Removed scope data.
 	 */
 
-	static removeEntitiesInScope(programId, entities) {
+	static removeEntitiesInScope(programId, bodyData, userDetails, organizations) {
```

Comment:
@MallanagoudaB Please recheck the code based on the comments I added for addEntitiesInScope function. then you can make same changes in solution helper as well